tests/Unit/Panels/StreetsPanelTest.php

```diff
@@ -2,27 +2,92 @@
 
 namespace Wearesho\GoogleAutocomplete\Yii\Tests\Unit\Panels;
 
+use GuzzleHttp;
 use Wearesho\GoogleAutocomplete\Yii\Panels\StreetsPanel;
-use Wearesho\GoogleAutocomplete\Yii\Tests\TestCase;
+use Wearesho\GoogleAutocomplete\Yii\Tests\PanelTestCase;
 
 /**
  * Class StreetsPanelTest
  * @package Wearesho\GoogleAutocomplete\Yii\Tests\Unit\Panels
  * @coversDefaultClass StreetsPanel
  * @internal
  */
-class StreetsPanelTest extends TestCase
+class StreetsPanelTest extends PanelTestCase
 {
+    protected const TESTED_PANEL = StreetsPanel::class;
+
     /** @var StreetsPanel */
-    protected $fakeStreetsPanel;
+    protected $panel;
+
+    protected function setUp()
+    {
+        parent::setUp();
+
+        $this->expectedRules = [
```

Comment:
Rules should not be checked as array equal. Rules should be checked using validation.
So, you need to fill attribute and then run ->validate(['attributeName']) with some error/success expectation.